### ISSUE TITLE
Python script for creating folders failed if root folder being open by Atom

### ISSUE THREAD

**Issue by chhlatnash** [atom/atom#8675]
I'm running a Python script inside a folder and that script will replace two folder, **bin** and **assets** folder, and create some file inside that two folders. The script mostly failed to create one of the two folders, sometimes it could create **bin** folder but failed to create **assets** or vice versa. The folder that was failed to create could be seen inside **Windows Explorer** but cannot be enter or delete but that folder will disappeared immediately after Atom was closed.

**Comment by lee-dohm** [comment]
I'm not sure what this has to do with Atom? Can you please clarify:

1. What version of Windows you're running?
1. Which version of Atom this is?
1. What the exact steps to reproduce are?
1. Can you reproduce the issue by performing the operations manually, in other words without the Python script?

Thanks

**Comment by chhlatnash** [comment]
1. I'm running Atom version 10.0.1 on Windows 8.1 Enterprise N, with all Nuclide packages installed.
2. I think this might caused by conflict between **nuclide-file-tree** and core package **tree-view**.
At first I thought one of them was disabled, but it did not, as I can only see one tree windows. After disable **tree-view** package the problem gone.
3. I have tried to create folders many time using Windows Explorer but there is no problems at all. But hen I run **cocos** command from **Cocos2d-x 3.7.1** using **Windows Command prompt or Windows Powershell** the problems arise. The command sometimes succeeded and sometimes failed but if it failed once there seem to be no chance of success unless I restart my Windows (PC).
4. Now I have another issue: **tree-view** package re-enabled automatically after **nuclide-file-tree** was update.
5. Another possible cause of **tree-view** package re-enabled automatically is Atom process won't close. Atom windows closed and disappeared but Atom process still exist when I checked in Task Manager. If I tried to open Atom again it show blank windows without loading project. When I terminate/end task of Atom process and start the Atom again I found out that all disable packages was re-enabled.
